commerce-admin-console/src/app/features/catalogs/components/header/header.component.html

```diff
@@ -0,0 +1,171 @@
+<a class="navbar-brand ml-3" href="#">
+    <span class="">Catalog</span>
```

Comment:
Translatable text should not be directly included. We need to consider localization. Right now we're putting all of the translatable text into src/assets/i18n/en.json. You can follow the existing patterns.